### PR TITLE
Create de.lua

### PR DESCRIPTION
**Pull request by Seraph1982**

### CODE REVIEW

File: locales/de.lua [hunk 1]
```diff
@@ -0,0 +1,134 @@
+local Translations = {
+    error = {
+        canceled = 'Abgebrochen',
+        bled_out = 'Du bist ausgeblutet...',
+        impossible = 'Aktion unmöglich...',
```
Comment (Tomboru) [inline locales/de.lua:5]:
```suggestion
        impossible = 'Aktion kann nicht ausgeführt werden...',
```

File: locales/de.lua [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        no_player = 'Kein Spieler in der nähe',
+        no_firstaid = 'Du benötigst ein First Aid Kit',
```
Comment (Tomboru) [inline locales/de.lua:7]:
```suggestion
        no_firstaid = 'Du benötigst ein Erste-Hilfe-Kasten',
```

File: locales/de.lua [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        no_bandage = 'Du benötigst einen Verband',
+        beds_taken = 'Betten sind belegt...',
```
Comment (Tomboru) [inline locales/de.lua:9]:
```suggestion
        beds_taken = 'Die Betten sind belegt...',
```

File: locales/de.lua [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        possessions_taken = 'Alle deine Besitztümmer wurden genommen...',
```
Comment (Tomboru) [inline locales/de.lua:10]:
```suggestion
        possessions_taken = 'Dir wurde alles abgenommen...',
```

File: locales/de.lua [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+        not_enough_money = 'Du hast nicht genügend Geld bei dir...',
+        cant_help = 'Du kannst dieser Person nicht helfen...',
+        not_ems = 'Du bist kein Rettungsdienst',
+        not_online = 'Spieler nicht online'
+    },
+    success = {
+        revived = 'du hast eine Person wiederbelebt',
```
Comment (Tomboru) [inline locales/de.lua:17]: ### 
```suggestion
        revived = 'Du hast eine Person wiederbelebt',
```

File: locales/de.lua [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+        healthy_player = 'Spieler ist gesund',
+        helped_player = 'Du hast der Person geholfen',
+        wounds_healed = 'Deine Wunden wurden geheilt!',
```
Comment (Tomboru) [inline locales/de.lua:20]:
```suggestion
        wounds_healed = 'Deine Wunden wurden verarztet!',
```

File: locales/de.lua [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+        being_helped = 'Dir wurde geholfen...'
+    },
+    info = {
+        civ_died = 'Zivilist verstorben',
+        civ_down = 'Zivilist Down',
```
Comment (Tomboru) [inline locales/de.lua:25]:
```suggestion
        civ_down = 'Zivilist verletzt',
```

File: locales/de.lua [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+        civ_call = 'Zivilisten Anruf',
+        self_death = 'Er selbst oder ein NPC',
```
Comment (Tomboru) [inline locales/de.lua:27]:
```suggestion
        self_death = 'Sie selbst oder ein NPC',
```

File: locales/de.lua [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+        wep_unknown = 'Unbekannt',
+        respawn_txt = 'Wiederbelebung in: ~r~%{deathtime}~s~ Sekunden',
```
Comment (Tomboru) [inline locales/de.lua:29]:
```suggestion
        respawn_txt = 'RESPAWNE IN: ~r~%{deathtime}~s~ SEKUNDEN',
```

File: locales/de.lua [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+        respawn_revive = 'Halte [~r~E~s~] für %{holdtime} Sekunden um Wiederbelebt zu werden zu Preis von $~r~%{cost}~s~',
```
Comment (Tomboru) [inline locales/de.lua:30]:
```suggestion
        respawn_revive = 'HALTE [~r~E~s~] FÜR %{holdtime} SEKUNDEN UM ZU RESPAWNEN FÜR $~r~%{cost}~s~',
```

File: locales/de.lua [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+        bleed_out = 'Du blutest aus in: ~r~%{time}~s~ Sekunden',
```
Comment (Tomboru) [inline locales/de.lua:31]:
```suggestion
        bleed_out = 'DU BLUTEST AUS IN: ~r~%{time}~s~ SEKUNDEN',
```

File: locales/de.lua [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+        bleed_out_help = 'Du blutest aus in: ~r~%{time}~s~ Sekunden, dir kann geholfen werden',
```
Comment (Tomboru) [inline locales/de.lua:32]:
```suggestion
        bleed_out_help = 'DU BLUTEST AUS IN: ~r~%{time}~s~ SEKUNDEN, DIR KANN GEHOLFEN WERDEN',
```

File: locales/de.lua [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+        request_help = 'Drücke [~r~G~s~] für eine hilfeanfrage',
```
Comment (Tomboru) [inline locales/de.lua:33]:
```suggestion
        request_help = 'DRÜCKE [~r~G~s~] UM EIN SANITÄTER ZU RUFEN',
```

File: locales/de.lua [hunk 14]
```diff
@@ -0,0 +1,134 @@
...
+        help_requested = 'Rettungsdienst wurde verständigt',
```
Comment (Tomboru) [inline locales/de.lua:34]:
```suggestion
        help_requested = 'SANITÄTER WURDEN BENACHRICHTIGT',
```

File: locales/de.lua [hunk 15]
```diff
@@ -0,0 +1,134 @@
...
+        amb_plate = 'AMBU', -- Should only be 4 characters long due to the last 4 being a random 4 digits
+        heli_plate = 'LIFE',  -- Should only be 4 characters long due to the last 4 being a random 4 digits
+        status = 'Status Prüfung',
+        is_staus = 'Ist %{status}',
+        healthy = 'Du bist wieder ganz gesund!',
```
Comment (Tomboru) [inline locales/de.lua:39]:
```suggestion
        healthy = 'Du bist wieder völlig gesund!',
```

File: locales/de.lua [hunk 16]
```diff
@@ -0,0 +1,134 @@
...
+        safe = 'Krankenhaus Safe',
+        pb_hospital = 'Pillbox Krankenhaus',
+        pain_message = 'Dein %{limb} fühlt sich %{severity} an',
+        many_places = 'Du hast schmerzen an verschiedenen Stellen...',
+        bleed_alert = 'Du bist %{bleedstate}',
+        ems_alert = 'Rettungsdienst Alarm - %{text}',
+        mr = 'Herr.',
```
Comment (Tomboru) [inline locales/de.lua:46]:
```suggestion
        mr = 'Herr',
```

File: locales/de.lua [hunk 17]
```diff
@@ -0,0 +1,134 @@
...
+        mrs = 'Frau.',
```
Comment (Tomboru) [inline locales/de.lua:47]:
```suggestion
        mrs = 'Frau',
```

File: locales/de.lua [hunk 18]
```diff
@@ -0,0 +1,134 @@
...
+        dr_needed = 'Ein Doktor wird im Pillbox Krankenhaus benötigt',
```
Comment (Tomboru) [inline locales/de.lua:48]:
```suggestion
        dr_needed = 'Ein Arzt wird im Pillbox Krankenhaus benötigt',
```

File: locales/de.lua [hunk 19]
```diff
@@ -0,0 +1,134 @@
...
+        ems_report = 'Rettungsdienst Report',
+        message_sent = 'Nachricht zu senden',
+        check_health = 'Prüfe die Gesunheit eines Spielers',
+        heal_player = 'Heile einen Spieler',
+        revive_player = 'Belebe einen Spieler',
```
Comment (Tomboru) [inline locales/de.lua:53]:
```suggestion
        revive_player = 'Spieler wiederbeleben',
```

File: locales/de.lua [hunk 20]
```diff
@@ -0,0 +1,134 @@
...
+        revive_player_a = 'Belebe einen Spieler oder dich selbst (Admin Only)',
+        player_id = 'Spieler ID (may be empty)',
+        pain_level = 'Setzt dein oder eines Spielers schmerzlevel (Admin Only)',
```
Comment (Tomboru) [inline locales/de.lua:56]:
```suggestion
        pain_level = 'Stellen Sie Ihre oder die Schmerzstufe eines Spielers (Admin Only)',
```

File: locales/de.lua [hunk 21]
```diff
@@ -0,0 +1,134 @@
...
+        kill = 'Töte einen Spieler oder dich selbst (Admin Only)',
+        heal_player_a = 'Keile einen Spieler oder dich selbst (Admin Only)',
```
Comment (Tomboru) [inline locales/de.lua:58]:
```suggestion
        heal_player_a = 'Heile einen Spieler oder dich selbst (Admin Only)',
```

File: locales/de.lua [hunk 22]
```diff
@@ -0,0 +1,134 @@
...
+    },
+    mail = {
+        sender = 'Pillbox Krankenhaus',
+        subject = 'Krankenhauskosten',
+        message = 'Hallo %{gender} %{lastname}, <br /><br />Hiermit erhalten Sie eine Email über die Kosten ihres letzten Krankenhaus besuchs.<br />Die gesammt Kosten betragen: <strong>$%{costs}</strong><br /><br />Wir wünschen eine schnelle Genesung!'
```
Comment (Tomboru) [inline locales/de.lua:63]:
```suggestion
        message = 'Hallo %{gender} %{lastname}, <br /><br />Hiermit erhalten Sie eine Email für die Kosten ihres letzten Krankenhaus Besuchs.<br />Die Gesamtkosten betragen: <strong>$%{costs}</strong><br /><br />Wir wünschen eine schnelle Genesung!'
```

File: locales/de.lua [hunk 23]
```diff
@@ -0,0 +1,134 @@
...
+        painful = 'schmerzhaft',
+        really_painful = 'sehr schmerzhaft',
+        little_bleed = 'wenig blutend...',
+        bleed = 'blutend...',
+        lot_bleed = 'etwas mehr blutend...',
+        big_bleed = 'sehr stark blutend...',
+    },
+    menu = {
+        amb_vehicles = 'Rettungsdienst Fahrzeuge',
+        close = '⬅ Schließe Menu',
```
Comment (Tomboru) [inline locales/de.lua:77]:
```suggestion
        close = '⬅ Menü Schließen',
```

File: locales/de.lua [hunk 24]
```diff
@@ -0,0 +1,134 @@
...
+    },
+    text = {
+        pstash_button = '~g~E~w~ - Persönliches Fach',
```
Comment (Tomboru) [inline locales/de.lua:80]:
```suggestion
        pstash_button = '~g~E~w~ - Persönliches Lager',
```

File: locales/de.lua [hunk 25]
```diff
@@ -0,0 +1,134 @@
...
+        pstash = 'Persönliches Fach',
```
Comment (Tomboru) [inline locales/de.lua:81]:
```suggestion
        pstash = 'Persönliches Lager',
```

File: locales/de.lua [hunk 26]
```diff
@@ -0,0 +1,134 @@
...
+        onduty_button = '~g~E~w~ - Dienst antretten',
+        offduty_button = '~r~E~w~ - Dienst quittieren',
+        duty = 'In/Ausser Dienst',
+        armory_button = '~g~E~w~ - Waffenschrank',
```
Comment (Tomboru) [inline locales/de.lua:85]:
```suggestion
        armory_button = '~g~E~w~ - Arsenal',
```

File: locales/de.lua [hunk 27]
```diff
@@ -0,0 +1,134 @@
...
+        armory = 'Waffenschrank',
```
Comment (Tomboru) [inline locales/de.lua:86]:
```suggestion
        armory = 'Arsenal',
```

File: locales/de.lua [hunk 28]
```diff
@@ -0,0 +1,134 @@
...
+        storeveh_button = '~g~E~w~ - Verwahre Fahrzeug',
```
Comment (Tomboru) [inline locales/de.lua:87]:
```suggestion
        storeveh_button = '~g~E~w~ - Fahrzeug einparken',
```

File: locales/de.lua [hunk 29]
```diff
@@ -0,0 +1,134 @@
...
+        veh_button = '~g~E~w~ - Fahrzeuge',
+        storeheli_button = '~g~E~w~ - Verwahre Helikopter',
```
Comment (Tomboru) [inline locales/de.lua:89]:
```suggestion
        storeheli_button = '~g~E~w~ - Helikopter einparken',
```

Even if its sounds weird, thats the official term

File: locales/de.lua [hunk 30]
```diff
@@ -0,0 +1,134 @@
...
+        heli_button = '~g~E~w~ - Nimm einen Helikopter',
```
Comment (Tomboru) [inline locales/de.lua:90]:
```suggestion
        heli_button = '~g~E~w~ - Helikopter ausparken',
```

File: locales/de.lua [hunk 31]
```diff
@@ -0,0 +1,134 @@
...
+        elevator_roof = '~g~E~w~ - Benutze Aufzug aufs Dach',
+        elevator_main = '~g~E~w~ - Benutze Aufzug runter',
+        bed_out = '~g~E~w~ - Um aus dem Bett zu steigen..',
+        call_doc = '~g~E~w~ - Rufe Doktor',
```
Comment (Tomboru) [inline locales/de.lua:94]:
```suggestion
        call_doc = '~g~E~w~ - Arzt rufen',
```

File: locales/de.lua [hunk 32]
```diff
@@ -0,0 +1,134 @@
...
+        left_leg = 'Linkes Bein',
+        left_foot = 'Linker Fuß',
+        right_arm = 'Rechter Arm',
+        right_hand = 'Rechte Hand',
+        right_fingers = 'Rechte Finger',
+        right_leg = 'Rechtes Bein',
+        right_foot = 'Rechter Fuß',
+    },
+    progress = {
+        ifaks = 'ifaks nehmen...',
```
Comment (Tomboru) [inline locales/de.lua:118]:
```suggestion
        ifaks = 'Benutze Erste-Hilfe-Kasten...',
```

File: locales/de.lua [hunk 33]
```diff
@@ -0,0 +1,134 @@
...
+        bandage = 'Verband benutzen...',
```
Comment (Tomboru) [inline locales/de.lua:119]:
```suggestion
        bandage = 'Benutze Verband...',
```

File: locales/de.lua [hunk 34]
```diff
@@ -0,0 +1,134 @@
...
+        painkillers = 'Schmerzmittel nehmen...',
```
Comment (Tomboru) [inline locales/de.lua:120]:
```suggestion
        painkillers = 'Nehme Schmerzmittel...',
```

File: locales/de.lua [hunk 35]
```diff
@@ -0,0 +1,134 @@
...
+        revive = 'Person wiederbeleben...',
```
Comment (Tomboru) [inline locales/de.lua:121]:
```suggestion
        revive = 'Wiederbelebe Person...',
```

**Review by Tomboru — CHANGES_REQUESTED** [review]
Just changed some weird translations.

They would make more sence in the context.

**Comment by FjamZoo** [comment]
Please update the changes proposed.

File: locales/de.lua (hunk 2 above)
Comment (MikesModZ-Modding) [reply locales/de.lua:7]: > 
**einen** Erste-Hilfe-Kasten

File: locales/de.lua (hunk 35 above)
Comment (MikesModZ-Modding) [reply locales/de.lua:121]: > 

Person reanimieren

**Pull request merged**